js/index.js

```diff
@@ -1,102 +1,116 @@
+/* eslint-disable max-classes-per-file */
 const bookTitle = document.getElementById('title');
 const bookAuthor = document.getElementById('author');
 const bookContainer = document.querySelector('.book-list');
 const form = document.getElementById('form');
 
-// booksArray to store array of object
-let books = [];
+class AwesomeBooks {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
 
-// Create a js Object to store form data
-const formData = {
-  title: '',
-  author: '',
-};
+class LocalStorage {
+  // eslint-disable-next-line class-methods-use-this
+  getLocalStorage() {
+    let books;
+    if (localStorage.getItem('books') == null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
 
-function removeElementFromUI(target) {
-  if (target.classList.contains('remove-btn')) {
-    target.parentElement.remove();
+  setStorage(book) {
+    const books = this.getLocalStorage();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
   }
-}
 
-function removeFromBooks(title) {
-  let tempBooks;
-  if (localStorage.getItem('books') === null) {
-    tempBooks = [];
-  } else {
-    tempBooks = JSON.parse(localStorage.getItem('books'));
-    tempBooks.forEach((book, index) => {
+  removeBookFromStorage(title) {
+    const books = this.getLocalStorage();
+    books.forEach((book, index) => {
       if (book.title === title) {
```

Comment:
- [ ] I created 4 books with the same title, then I remove one of the books but 2 books got deleted. I suggest you make use of a unique `id` to identify each book and use the filter method to remove a book with the id of the book to be deleted.